### PR TITLE
Write guide on how to make detekt silent

### PR DESCRIPTION
**Pull request by arturbosch**

### CODE REVIEW

File: docs/_posts/2019-08-14-custom-console-reports.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+detekt's reporting mechanism relies on implementations of [ConsoleReport](https://arturbosch.github.io/detekt/pages/kdoc/detekt-api/io.gitlab.arturbosch.detekt.api/-console-report/index.html)'s.
+The cli module and therefore the Gradle plugin implement a bunch of this reports.
+
+A typical detekt report will look like following:
+
+![report](images/howto-silent-run/typical_console_report.png)
+
+There are many different parts which might or might not interest you.
+If one part is not important to you, it can be excluded in the yaml configuration file.
+A _silent_ configuration would exclude all possible processors and reports:
```
Comment (schalkms) [inline docs/_posts/2019-08-14-custom-console-reports.md]: I would format **silent** bold.

File: docs/_posts/2019-08-14-custom-console-reports.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    - 'BuildFailureReport'
+```  
+
+Running with this config won't produce any console messages: 
+
+![report](images/howto-silent-run/silent_run.png)
+
+Just verify that the `./report.txt` is not empty ;).
+
+We mind find detekt's `FindingsReport` too verbose and just want to print one line of message per finding.
```
Comment (schalkms) [inline docs/_posts/2019-08-14-custom-console-reports.md]: might instead of mind

Comment (schalkms) [inline docs/_posts/2019-08-14-custom-console-reports.md]: print one message line per finding

File: docs/_posts/2019-08-14-custom-console-reports.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+This can be archived by implementing a `ConsoleReport`.
```
Comment (schalkms) [inline docs/_posts/2019-08-14-custom-console-reports.md]: achieved insteaf of archived

Comment (schalkms) [inline docs/_posts/2019-08-14-custom-console-reports.md]: by implementing a **custom** `ConsoleReport`

**Comment by arturbosch** [comment]
Your right.
We could 
1. split news and guides, introducing a new navigation item called `guides` next to news.
2. Just link the guides tag in the readme https://arturbosch.github.io/detekt/tag_guides.html
3. Rename `news` to `blog`

**Pull request merged**